tests/integration/test_vulnerability_detector/test_feeds/test_invalid_values_redhat_feeds.py

```diff
@@ -0,0 +1,151 @@
+# Copyright (C) 2015-2020, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+import os
+import pytest
+import json
+from time import sleep
+import itertools
+
+from wazuh_testing.tools import LOG_FILE_PATH
+from wazuh_testing.tools.configuration import load_wazuh_configurations
+from wazuh_testing.tools.monitoring import FileMonitor
+from wazuh_testing.tools.file import truncate_file, read_file, write_file
+from wazuh_testing.vulnerability_detector import CUSTOM_NVD_VULNERABILITIES_1, CUSTOM_NVD_FEED, \
+                                                 REDHAT_KEY_FIELDS_FEEDS, set_custom_system, \
+                                                 check_vulnerability_event, check_vulnerabilities_number, \
+                                                 clean_vuln_and_sys_programs_tables
+from wazuh_testing.tools.services import control_service
+
+# Marks
+pytestmark = pytest.mark.tier(level=1)
```

Comment:
It would be interesting to move these tests to tier 2 due to the time that it takes to be completed.

Reply:
Done in ab1c229f9625689d173018ae41d5437228ded6be